travis_packages.sh

```diff
@@ -15,6 +15,7 @@ b libvita2d
 b libmad
 b libogg
 b libvorbis
+b flac
```

Comment:
Add it [there](https://github.com/vitasdk/vdpm/blob/master/install-all.sh) as well.